libs/template/presets/src/features/fulfillment/app.ts

```diff
@@ -31,6 +32,7 @@ export function fulfillmentFeatures(
     {
       resources: fulfillmentResources,
     },
+    new PushNotificationFeature(),
```

Comment:
We should use WebPushNotification feature in the fulfillment app
 ```suggestion
    new WebPushNotificationFeature(),
```